libbpf-tools/bashreadline.bpf.c

```diff
@@ -0,0 +1,35 @@
+// SPDX-License-Identifier: GPL-2.0
+// Copyright (c) 2020 Wenbo Zhang
```

Comment:
You should update this copyright statement.

Reply:
And use C style comment.

Reply:
Both lines in C style?